src/Microsoft.AspNetCore.Localization/RequestCultureProvider.cs

```diff
@@ -13,6 +13,12 @@ namespace Microsoft.AspNetCore.Localization
     public abstract class RequestCultureProvider : IRequestCultureProvider
     {
         /// <summary>
+        /// Result that indicates that this instance of <see cref="RequestCultureProvider" /> could not determine the
+        /// request culture.
+        /// </summary>
+        protected static readonly Task<ProviderCultureResult> NullResult = Task.FromResult(default(ProviderCultureResult));
```

Comment:
This removes the last use of `Task<T>` from our code base (and we can remove the package). Given the pattern requires returning this, it made sense to put the field on the base type. I can alternatively make it an `internal` field or put it in an `.Internal` type.